cmd/xk6/main.go

```diff
@@ -30,123 +29,66 @@ import (
 )
 
 var (
-	k6Version    = os.Getenv("K6_VERSION")
-	k6Repo       = os.Getenv("XK6_K6_REPO")
-	raceDetector = os.Getenv("XK6_RACE_DETECTOR") == "1"
-	skipCleanup  = os.Getenv("XK6_SKIP_CLEANUP") == "1"
+	errExpectedValue  = fmt.Errorf("expected value")
+	errInvalidValue   = fmt.Errorf("invalid value")
+	errMissingFlag    = fmt.Errorf("missing flag")
+	errMissingReplace = fmt.Errorf("missing replace")
 )
 
+type BuildOps struct {
+	K6Version      string
+	Extensions     []string
+	Replacements   []string
+	OutFile        string
+	OutputOverride bool
+}
+
 func main() {
+	log := slog.New(
```

Comment:
The log output is very different from the original.

log := slog.Default()

This way the output would be more similar to the original output (except that the log level is not in parentheses).